database/build/last_repo_test.go

```diff
@@ -32,6 +32,7 @@ func TestBuild_Engine_LastBuildForRepo(t *testing.T) {
 	_repo.SetAllowEvents(api.NewEventsFromMask(1))
 	_repo.SetPipelineType(constants.PipelineTypeYAML)
 	_repo.SetTopics([]string{})
+	_repo.SetInstallID(0)
```

Comment:
I'm somewhat sure that you can remove all these setters for install ID if you add the field to the testutils.APIRepo function

Reply:
you are a legend 🙏 like 50 changed files removed